static/js/tools/map/bq_query_utils.ts

```diff
@@ -70,17 +71,21 @@ function getLatestObsDateView(
   return (
     `WITH ${LATEST_OBS_DATE_VIEW_NAME} AS (` +
     `
-  ${getChildPlaceView(place)}
-  SELECT O.variable_measured as StatVarId,
-        O.observation_about as PlaceId,
-        MAX(O.Observation_date) as LatestDate
-  FROM \`data_commons.Observation\` as O
-  JOIN ${CHILD_PLACE_VIEW_NAME} ON TRUE
-  WHERE ${getSvMetadataPredicate("O", metadata)} AND
-        O.variable_measured = '${sv.dcid}' AND
-        O.observation_about = ${CHILD_PLACE_VIEW_NAME}.PlaceId
-  GROUP BY StatVarId, PlaceId
-)`
+    ${getChildPlaceView(place)}
+    )
+    SELECT O.variable_measured as StatVarId,
+           O.observation_about as PlaceId,
+           MAX(O.Observation_date) as LatestDate
+    FROM \`data_commons.Observation\` as O
+    JOIN ${CHILD_PLACE_VIEW_NAME} ON TRUE
+    JOIN \`data_commons.Provenance\` as I ON TRUE
+    WHERE O.prov_id = I.prov_id AND
+          O.variable_measured = '${sv.dcid}' AND
+          O.observation_about = ${CHILD_PLACE_VIEW_NAME}.PlaceId AND
+          ${getSvMetadataPredicates("O", "I", metadata).join(
+            " AND\n          "
```

Comment:
i'm curious why there's so many trailing spaces after the newline. if it's to maintain the indentation, could you leave a comment?

Reply:
done